.gitignore

```diff
@@ -5,4 +5,5 @@
 /Tests/Controller/app/logs
 /var/
 .phpunit.result.cache
-.php-cs-fixer.cache
\ No newline at end of file
+.php-cs-fixer.cache
+.idea
```

Comment:
You can add this to your local global .gitignore file, and you won't have to bother anyone about the fact that you're using IntelliJ.

```
> cat ~/.gitignore
.idea/
```

Reply:
true but that also helps everyone that do not have it 🤷

Reply:
For me, this line is unrelated to this project.

If you think it belongs here, it would mean that you need to add this to all the projects that you contribute to.

Reply:
don't we add /vendor/ top all the PHP projects 🤷